codecov.yml

```diff
@@ -2,8 +2,11 @@ coverage:
   status:
     project:
       default:
+        informational: true
```

Comment:
Prevents ❌ in CI in cases like:

<img width="866" alt="Screenshot 2022-10-31 at 19 02 09" src="https://user-images.githubusercontent.com/608862/199088832-1e451dd4-c311-47b0-a6f3-5f3f42739962.png">

This will be reverted when coverage stabilises.